services/concept_insights/v2.js

```diff
@@ -175,13 +173,13 @@ module.exports = function (RED) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['account_id' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2398643202/issues/source?bid=3225115&fileBranchId=3230627#l173)